### ISSUE TITLE
java.lang.IllegalArgumentException: the bind value at index 1 is null

### ISSUE THREAD

**Issue by faldujaldeep** [urbanairship/android-library#173]
I am getting below exception while customizing notification using NotificationProvider I am using latest SDK version 14.1.1 

2021-02-15 10:50:03.565 31533-32604/my.yes.yes4g E/AndroidRuntime: FATAL EXCEPTION: UrbanAirship#7
    Process: my.yes.yes4g, PID: 31533
    java.lang.IllegalArgumentException: the bind value at index 1 is null
        at android.database.sqlite.SQLiteProgram.bindString(SQLiteProgram.java:183)
        at android.database.sqlite.SQLiteProgram.bindAllArgsAsStrings(SQLiteProgram.java:219)
        at android.database.sqlite.SQLiteDirectCursorDriver.query(SQLiteDirectCursorDriver.java:49)
        at android.database.sqlite.SQLiteDatabase.rawQueryWithFactory(SQLiteDatabase.java:1959)
        at android.database.sqlite.SQLiteDatabase.queryWithFactory(SQLiteDatabase.java:1806)
        at android.database.sqlite.SQLiteDatabase.query(SQLiteDatabase.java:1677)
        at android.database.sqlite.SQLiteDatabase.query(SQLiteDatabase.java:1883)
        at com.urbanairship.util.DataManager.query(DataManager.java:315)
        at com.urbanairship.util.DataManager.query(DataManager.java:303)
        at com.urbanairship.push.notifications.NotificationChannelRegistryDataManager.getChannel(NotificationChannelRegistryDataManager.java:124)
        at com.urbanairship.push.notifications.NotificationChannelRegistry$1.run(NotificationChannelRegistry.java:89)
        at com.urbanairship.util.SerialExecutor$1.run(SerialExecutor.java:41)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
        at com.urbanairship.util.AirshipThreadFactory$1.run(AirshipThreadFactory.java:50)
        at java.lang.Thread.run(Thread.java:919)

I am using the same code that is available here https://docs.airship.com/platform/android/push-notifications/#custom-notification-provider

I am trying to send a simple notification from the dashboard and the app getting crash.

**Comment by rlepinski** [comment]
Thanks for the report. We will fix this crash in 14.2.0 that should be coming out this week. You can work around this issue by supplying the notification channel on the builder:
```
        NotificationCompat.Builder builder = new NotificationCompat.Builder(context, arguments.getNotificationChannelId())
```

We will fix the code sample as well.

**Comment by rlepinski** [comment]
Fixed in 14.2.0
